README.md

```diff
@@ -1 +1,82 @@
-# kinesis-trigger
+# kafka-trigger
```

Comment:
I would point here to the docs in the kubeless repo to avoid having the documentation in different repositories (which would make it more difficult to maintain the docs in kubeless.io/docs)